frontend/src/pages/[lang]/profile/edit/index.tsx

```diff
@@ -82,10 +82,9 @@ export const EditProfilePage: React.FC = () => {
   const router = useRouter();
   const language = useCurrentLanguage();
   const [loggedIn] = useLoginState();
-  const [
-    formState,
-    { text, select, checkbox, raw },
-  ] = useFormState<MeUserFields>(
+  const [formState, { text, select, checkbox, raw }] = useFormState<
+    MeUserFields
+  >(
```

Comment:
prettier misconfiguration between pre-commit and local prettier

we need to look at it and fix it